.devcontainer/docker-compose.yml

```diff
@@ -100,10 +105,13 @@ services:
                 aliases:
                     - keycloak.local
 
-    memcached:
-        container_name: memcached_${PROJ_PREFIX:-myproj}
-        image: memcached:latest
+    redis:
+        container_name: redis
+        image: redis:latest
         restart: always
+        command: redis-server --requirepass ${REDIS_PASSWORD:-password}
+        ports:
+            - ${REDIS_PORT:-6379}:6379
```

Comment:
Do we need to expose this on the host?
I don't think so, but if we do, then we should add it to the `.env.example`

Reply:
I was using a tool to inspect Redis so I think that's why I exposed it. Doesn't need to be exposed.